stdlib/SuiteSparse/src/umfpack.jl

```diff
@@ -428,57 +428,208 @@ for itype in UmfpackIndexTypes
                            lnz, unz, n_row, n_col, nz_diag, lu.numeric)
             (lnz[], unz[], n_row[], n_col[], nz_diag[])
         end
-        function umf_extract(lu::UmfpackLU{Float64,$itype})
+        function umf_extract(lu::UmfpackLU{Float64,$itype}, d::Symbol)
```

Comment:
```suggestion
        function umf_extract(lu::UmfpackLU{Float64,$itype}) = umf_extract(lu, :)
        function umf_extract(lu::UmfpackLU{Float64,$itype}, d::Symbol)
```
for backward compatibility? I guess it's not exported, but maybe people still use it. Well, or what about my other suggestion below?